js/navbar.js

```diff
@@ -1,12 +1,17 @@
-var lastId,
+var lastScrollId,
+    lastSubId,
     topMenu = $("#navOptions"),
     topMenuHeight = topMenu.outerHeight(),
     // All list items
     menuItems = topMenu.find("a"),
```

Comment:
I believe there are more `<a>`- html tags than just the menuItems. For the sake of readability and possible creation of bugs at a later point, I might add a class to the `<a>` tags.